src/twisted/mail/test/test_pop3client.py

```diff
@@ -586,7 +595,8 @@ def cbDisconnected(ign):
                 ['CAPA', 'STLS', 'CAPA', 'QUIT'])
```

Comment:
wouldn't input be bytes? And therefore `[b'CAPA', ...`

Reply:
Good point.